src/pages/Maxim/MaximTask1/MaximTask1.tsx

```diff
@@ -4,26 +4,51 @@ import { Link } from 'react-router-dom'
 
 // Здесь импортируется Модульный CSS
 import css from './MaximTask1.module.css'
+import React from 'react'
 
 // 'это реакт компонент, доступный по адресу (роуту) /maxim/first-task'. Компонент объявлен как функция JS, возаращая JSX.
 // Этот компонент (функция) может принимать пропсы (аргументы) - для того чтобы работать с ними.
 export const MaximTask1 = () => {
   // В теле функции обычно пишется логика, которая понадобится для отображения. Мы пока это не трогаем.
-
+ 
   // Возвращает компонент специальный синтаксис: JSX. Он очень похож на HTML, но у него побольше возможностей.
   // Пока можешь просто учиться верстать на JSX
   return (
     <div>
       <p>
         <Link to={'/'}>Home</Link>
       </p>
-
-      <p>
+      <h1>ПУТЯМИ ДОБРОТЫ</h1>
+      <div className={css['myText']}>
```

Comment:
Вот такая запись короче и нагляднее. Рекомендую использовать ее. И здесь, и в остальных местах.

```suggestion
      <div className={css.myText}>
```